coordinator/src/db/user.rs

```diff
@@ -160,3 +161,22 @@ pub fn get_users(conn: &mut PgConnection, trader_ids: Vec<PublicKey>) -> Result<
 
     Ok(users)
 }
+
+pub fn check_version(conn: &mut PgConnection, trader_id: &PublicKey) -> Result<()> {
+    let user: User = users::table
+        .filter(users::pubkey.eq(trader_id.to_string()))
+        .first(conn)?;
+
+    let app_version = match user.version {
+        Some(version) => version,
+        None => bail!("No version found"),
+    };
+
+    let coordinator_version = env!("CARGO_PKG_VERSION").to_string();
+    ensure!(
+        app_version == coordinator_version,
```

Comment:
:wrench: What if we use semver here? Then we can deploy patch fixes to the coordinator without breaking trading for our users.

Reply:
Fair enough. I just wanted to mention it.